demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -318,19 +318,23 @@ def remove_punctuation(word):
     def check_word(self, word):
         """Check if a word is legal"""
         # check camel cases
+        word = self.remove_punctuation(word)
+        sub_words = []
+        if '-' in word:
+            sub_words.extend(word.split('-'))
         if not self.no_camel_case and self.is_camel_case(word):
```

Comment:
`elif`?

Reply:
I wasn't too sure about the validity of this case, but let's go with elif.